### ISSUE TITLE
Add custom augmentations option to MM module

### ISSUE THREAD

**Issue by KirderfEman** [autogluon/autogluon#2795]
Today there are only some of the common augmentations techniques available in the MM module for vision tasks. 
Can you implement an option to the predictor function so the user can add additional torchvision aug. techniques. 

Here is an example how doing it manually:

In the file multimodal/data/process_image.py one adds the custom aug:

```python3
            elif trans_mode == "rrotation":
                processor.append(transforms.RandomRotation(degrees=(0,10),interpolation=InterpolationMode.NEAREST))
```

And then one can call it from here in the predictor:

```python3
                              "model.timm_image.train_transform_types": "[rrotation]",
                               "model.timm_image.val_transform_types": "[rrotation]"
```

Can you implement an option so the user can add the e.g. code directly to the process_image.py codes without doing it manually.
For example that the predictor look for additional custom user added torchvision aug. code in the predictor if not None, and if so add it to the rest.

Thanks!

**Comment by sxjscience** [comment]
Thanks for the suggestion. This is something we can do. CC @lzcemma @zhiqiangdon 

We tried internally and found that TrivialAugment seems to be pretty good in our internal benchmark. (Meanwhile, we are also doing research in data augmentation and has a paper in ICLR 2023: https://arxiv.org/abs/2212.14453 , of which we plan to add to the repository.)

**Comment by KirderfEman** [comment]
Thanks, I think it is important that the more advanced user can choose and change custom aug. depending on the problem/image e.g. medical images. Sometimes one wants to try and test different setups.

Interesting with new paper, I will check it!

**Comment by zhiqiangdon** [comment]
Addressed by #3022
